features/User_can_see_the_campaigns_on_Index_page.feature

```diff
@@ -10,7 +10,7 @@ Feature: User can see the campaigns on index page
         |Brain surgery | I grew up always helping others. | 1000.00       |
      
         And I visit the site
-        Then I should see "Brain surgery"
-        And I should see "I grew up always helping others."
-        And I should see "1000"
-        And show me the page
\ No newline at end of file
+        Then I should see 'Brain surgery'
+        And I should see 'I grew up always helping others.'
+        And I should see '1000'
+      
```

Comment:
Why do you add this feature to this PR? It is out of scope for this user story, you should add it to description of this PR

Reply:
this should be here because its checking the output of the creation